specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -103,6 +103,20 @@ properties:
                     type: string
                     nullable: true
                     example: 'Provider Converted Advice and Guidance, Admin to Refer'
+              - type: object
```

Comment:
Remove this as included as part of cancel Appointment endpoint modelling (PR-155)

Reply:
Removed